repositories/repositories.bzl

```diff
@@ -219,9 +219,9 @@ def ros2_repositories():
         http_archive,
         name = "ros2",
         build_file = "@com_github_mvukov_rules_ros2//repositories:ros2.BUILD.bazel",
-        sha256 = "6e2611392776fdf39e47205db44956b77727cd561245c012ccc3ff5b1a4cd7ad",
-        strip_prefix = "ros2-release-humble-20231122",
-        urls = ["https://github.com/ros2/ros2/archive/refs/tags/release-humble-20231122.tar.gz"],
```

Comment:
Looks like this was not updated properly last time and running `//repositories/private:resolver` again would have downgraded to 20231122. `ros2_repositories_impl.bzl` was properly at 20240119, though, so most probably no harm done.

Reply:
Thanks for fixing this.